### PR TITLE
Propose list of interview related resources

### PR DESCRIPTION
**Pull request by jcsmileyjr**

### CODE REVIEW

**Comment by jcsmileyjr** [comment]
@rjbeers 
Question 1: I think I failed the lint because of the TOC (Table of Content) check. Was I suppose to update the table of content or not touch it (instructions in the readme, as I read them, said not to edit that portion). 

Thank you

**Comment by rjbeers** [comment]
I'll need to update the instructions to help someone adding a new section.

**Comment by jcsmileyjr** [comment]
Ok. Is there anything I need to do today?

File: README.md [hunk 1]
```diff
@@ -64,3 +64,15 @@
 ## React
 - [React Handbook by Faviocopes](https://www.freecodecamp.org/news/the-react-handbook-b71c27b0a795/) - Free fantastic e-book teaching you how to code with React.
 - [JavaScripts basics before learning React](https://dev.to/tracycss/the-vanilla-javascript-basics-to-know-before-learning-react-js-53aj) - Series of blog posts about JavaScript basics needed before learning React.
+
+## Technical Interview Study Material
```
Comment (rjbeers) [inline README.md]: Please review https://github.com/jcsmileyjr/awesome-code-connector/pull/1 and let me know if you are opposed to those changes.

**Comment by jcsmileyjr** [comment]
I reviewed and approve

**Review by rjbeers — APPROVED** [review]

**Pull request merged**